sesje/2015-02-11/reminder.md

```diff
@@ -0,0 +1,22 @@
+Przypomnienie:
+
+Instrukcja warunkowa `if` i jej warianty:
+```python
+name = input("Podaj swoje imię:")
+if name == "John":
+    print("Hello John! How are you?")
+elif name == "Kasia":
+    print("Witaj Kasia! Jak się masz?")
+else:
+    print("Hola extraño! Cómo estás?")  # Google translate (TM), Don't hate.
+```
+
+Pętla `while`:
+```python
+player_choice = "c"
+while player_choice == "c":
+    print("Game on!")
+    # ...
+    print("Do you want to play again? c - continue, q - quit")
```

Comment:
Chodziło mi o linię z komunikatem oraz warunek w `while` (na zakładce "Files changed" widać to "lepiej") - chodzi przede wszystkim o zwrócenie uwagi na tzw. "kodowanie życzeniowe" ;), że komunikat to jedno a kod, który ma działać dokładnie tak jak opisuje go komunikat to co innego.

Ale sam plik jako całość - jak najbardziej OK.